app/router.js

```diff
@@ -29,9 +38,35 @@ module.exports = app => {
       return result
     })
   })
+
+  app.use(offenderRoute, getOffenderDetails)
+  app.get(`${offenderRoute}/plans`, (req, res) => sentencePlanSummary(req, res))
+
+  // sentence plans summary
+  app.get([offenderRoute, `${offenderRoute}/plans`], sentencePlanSummary)
+
+  // diversity
+  app.get([`${offenderRoute}/edit-plan/:planid(\\d{1,})/diversity`], getDiversity)
+  app.post(
+    [`${offenderRoute}/edit-plan/:planid(\\d{1,})/diversity`],
+    diversityValidationRules(),
```

Comment:
This isn't really middleware, as such. Its a utility function, so probably doesn't sit here? 🤔